hail/python/hail/docs/datasets.rst

```diff
@@ -0,0 +1,41 @@
+.. _sec-datasets:
+
+:tocdepth: 1
+
+This page describes genetic datasets that have already been imported into
+Hail-friendly formats and are available for use through Hail's
+:meth:`.load_dataset` function.
+
```

Comment:
if we're going to have a top-level link to this page (which I support), then we need a warning here about how it's experimental and can change at any time.